clouddriver-integration/src/test/java/com/netflix/spinnaker/clouddriver/MySqlContainerTest.java

```diff
@@ -58,12 +61,12 @@ static void setup() throws Exception {
         .start();
 
     Slf4jLogConsumer logConsumer = new Slf4jLogConsumer(logger);
-    clouddriverContainer.start();
     clouddriverContainer.followOutput(logConsumer);
+    clouddriverContainer.start();
```

Comment:
Did you run into a scenario where having followOutput before start makes a difference?

Reply:
No, but it's a configuration and I found all examples on google using this way, so changed the order.